rust-symcrypt/BUILD.md

```diff
@@ -47,18 +49,18 @@ During runtime, Windows will handle finding all needed `dll`'s in order to run t
 
 For more info please see: [Dynamic-link library search order](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order)
 
-Here are 4 recommended options to ensure your `symcrypttestmodule.dll` is found by Windows during runtime.
+Here are 4 recommended options to ensure your `symcrypt.dll` is found by Windows during runtime.
 
-1. Put the symcrypttesmodule.dll in the same folder as your output `.exe` file. If you are doing development (not release), the common path will be: `C:\your-project\target\debug\`.
-2. Add the symcrypttestmoudle.dll path as a one time environment variable. 
+1. Put the `symcrypt.dll` in the same folder as your output `.exe` file. If you are doing development (not release), the common path will be: `C:\your-project\target\debug\`.
+2. Add the `symcrypt.dll` path as a one time environment variable. 
     ```powershell
     $env:PATH = "C:\Your-Path-To-SymCrypt\SymCrypt\bin\exe;$env:PATH"
     ```
     **Note:** This change will only persist within the current process, and you must re-set the PATH environment variable after closing the PowerShell window.
 
-3. Manually copy `symcrypttestmodule.dll` into your `C:/Windows/System32/` 
-    Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypttestmodule.dll`
-4. Permanently add the `symcrypttestmodule.dll` path into your System PATH environment variable. Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypttestmodule.lib`
+3. Manually copy `symcrypt.dll` into your `C:/Windows/System32/` 
+    Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.dll`

```

Comment:
We probably shouldn't recommend this to external developers either, especially given that future versions of Windows will include symcrypt.dll. For third party applications, the best practice is to either rely on the in-box binary (once it's available), or ship the DLL with your application so that it will be loaded from the application directory. The latter also allows the developer to guarantee that the version of the DLL matches what their application expects.

Reply:
I think in the case of shipping the DLL with application makes sense, but we still need the lib to be generated and linked to.

Maybe this can be done through the Bindgen somehow. I will do some investigation. 

Will follow up offline as well